requirements.txt

```diff
@@ -8,3 +9,4 @@ ensembl_prodinf_webhelp@git+https://github.com/Ensembl/ensembl-prodinf-webhelp.g
 ensembl_prodinf_masterdb@git+https://github.com/Ensembl/ensembl-prodinf-masterdb.git#egg=ensembl_prodinf_masterdb
 ensembl_prodinf_jira@git+https://github.com/Ensembl/ensembl-prodinf-jira.git#egg=ensembl_prodinf_jira
 mysqlclient~=2.0.3
+gunicorn~=20.1.0
```

Comment:
Imho, gunicorn shouldn't be a "direct" dependency, since it's only use in the context of Docker here.
I mean someone could want to run their app with another app than gunicorn: uWSGI, hypercorn etc...
Hence declared in a separated requirements file.

Reply:
hoo ok i will change it